Xrpl.C/Ripple.Binary.Codec/Types/Uint64.cs

```diff
@@ -9,33 +10,16 @@ public class Uint64 : Uint<ulong>
         public Uint64(ulong value) : base(value)
         {
         }
-        public override byte[] ToBytes()
-        {
-            return Bits.GetBytes(Value);
-        }
+        public override byte[] ToBytes() => Bits.GetBytes(Value);
 
-        public override string ToString()
-        {
-            return B16.Encode(ToBytes());
-        }
+        public override string ToString() => B16.Encode(ToBytes());
 
-        public static Uint64 FromJson(JToken token)
-        {
-            return Bits.ToUInt64(B16.Decode(token.ToString()), 0);
-        }
-        public static implicit operator Uint64(ulong v)
-        {
-            return new Uint64(v);
-        }
+        public static Uint64 FromJson(JToken token) => Bits.ToUInt64(B16.Decode(token.ToString()), 0);
 
-        public override JToken ToJson()
-        {
-            return ToString();
-        }
+        public static implicit operator Uint64(ulong v) => new Uint64(v);
 
-        public static Uint64 FromParser(BinaryParser parser, int? hint=null)
-        {
-            return Bits.ToUInt64(parser.Read(8), 0);
-        }
+        public override JToken ToJson() => ToString();
+
+        public static Uint64 FromParser(BinaryParser parser, int? hint = null) => Bits.ToUInt64(parser.Read(8), 0); //todo hint for what?
```

Comment:
length.

Reply:
for what? simply not used.

Reply:
It's only used a few times but it is used to parse the binary.

Reply:
ok